src/test/python/test_VantiqSDKLive.py

```diff
@@ -603,7 +603,7 @@ async def check_crud_operations(self, client: Vantiq, skip_pretest_cleanup: bool
         assert isinstance(vr, VantiqResponse)
         assert vr.is_success
         rows = vr.body
-        assert len(rows) == 0
+        assert len(rows) == 1
```

Comment:
Did something fix this?

Reply:
I think that when I created the tests, the _self_ cluster wasn't around always.  So it didn't show up -- now it does.  That's all.